Dockerfile

```diff
@@ -8,6 +8,18 @@ ENTRYPOINT ["/sbin/tini", "--", "/radicale/docker/entrypoint.sh"]
 EXPOSE 5232
 VOLUME ["/etc/radicale", "/var/lib/radicale"]
 
+ARG VERSION
```

Comment:
The `ARG` should be in a layer somewhere above the `apk` calls.

Reply:
isn't it?

Reply:
Well, you set the `ARG`s with every build. So, the actual apk-calls (which are unparametrized), won't get cached at any time.